<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskViewer</class>
 <widget class="QWidget" name="TaskViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Диспетчер задач</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>391</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ObelixPro</family>
    </font>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ObelixPro</family>
    </font>
   </property>
   <property name="text">
    <string>обновить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
